cypress/support/commands.ts

```diff
@@ -66,3 +86,28 @@ Cypress.Commands.overwrite(
       )
   }
 )
+
+Cypress.Commands.add('waitForAmplitudeEvent', (eventName, options) => {
+  const { alias = '@analytics', timeout = 20000, shouldTimeout = false } = options ?? {}
+  const startTime = new Date().getTime()
```

Comment:
nit: is there a reason to use `Date` instead of `performance.now()` here?

Reply:
i don't think it makes sense to use `performance.now()` here. the timeout is for the cypress command itself, not time since page load